Mathlib/RingTheory/Congruence.lean

```diff
@@ -183,15 +184,14 @@ section add_mul
 
 variable [Add R] [Mul R] (c : RingCon R)
 
-instance : Add c.Quotient := inferInstanceAs (Add c.toAddCon.Quotient)
+instance instAdd : Add c.Quotient := AddCon.hasAdd c.toAddCon
+-- inferInstanceAs (Add c.toAddCon.Quotient)
```

Comment:
is there any advantage to changing this?

Reply:
It works. Otherwise it fails with the change. That was the start of the discussion on Zulip